### PR TITLE
Devenv:LDAP: couple simplifications for LDAP

### PR DESCRIPTION
**Pull request by markelog**
* Add LDAP config instead sed use

* Add container name

* Add SizeLimit option to client and to server.
  Probably useless at this point, but it's better to have it then otherwise

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/bradfitz/gomemcache v0.0.0-20180710155616-bc664df96737
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/codegangsta/cli v1.20.0
+	github.com/davecgh/go-spew v1.1.1
```
Comment (xlson) [inline go.mod]: Doesn't feel like this changte belongs here?

File: pkg/services/ldap/ldap.go [hunk 2]
```diff
@@ -261,6 +261,7 @@ func (server *Server) getSearchRequest(
 	return &ldap.SearchRequest{
 		BaseDN:       base,
 		Scope:        ldap.ScopeWholeSubtree,
+		SizeLimit:    1000,
```
Comment (xlson) [inline pkg/services/ldap/ldap.go]: Is there any point in making this change without also implementing support for splitting up search request?

File: go.mod (hunk 1 above)
Comment (markelog) [reply go.mod]: We already use `spew` dep in production build - https://github.com/grafana/grafana/blob/0833f26b4337089af99b1b731d7e19588333e061/pkg/services/ldap/ldap.go#L182 and https://github.com/grafana/grafana/blob/0833f26b4337089af99b1b731d7e19588333e061/go.sum#L27. But we are missing it `go.mod`.

In other words that's the product of running `go mod tidy`

File: pkg/services/ldap/ldap.go (hunk 2 above)
Comment (markelog) [reply pkg/services/ldap/ldap.go]: As I mentioned in the description - no, not really. But at least it would explicitly works for 1000 users

**Review by xlson — APPROVED** [review]

**Pull request merged**